packages/client/src/runtime/core/engines/data-proxy/errors/utils/responseToError.test.ts

```diff
@@ -1,13 +1,9 @@
-import Debug from '@prisma/debug'
-import stripAnsi from 'strip-ansi'
-
-import { getErrorMessageWithLink } from '../common/errors/utils/getErrorMessageWithLink'
-import { responseToError } from '../data-proxy/errors/utils/responseToError'
-import type { RequestResponse } from '../data-proxy/utils/request'
+import type { RequestResponse } from '../../utils/request'
+import { responseToError } from './responseToError'
 
 const response = (body: string, code?: number, requestId?: string): RequestResponse => ({
   json: () => Promise.resolve(body),
-  text: () => body,
+  text: () => Promise.resolve(body),
```

Comment:
This method had incorrect return type, but since `engines-core` tests were not typechecked it was not caught.